src/Meta/Metadata.php

```diff
@@ -46,47 +46,48 @@ final class Metadata implements MetadataInterface
     private $options;
 
     /**
-     * @param string               $title
-     * @param string               $description
-     * @param string|null          $image
-     * @param string|null          $domain
      * @param array<string, mixed> $options
      */
-    public function __construct($title, $description = null, $image = null, $domain = null, array $options = [])
-    {
+    public function __construct(
+        string $title,
+        string $description = null,
```

Comment:
```suggestion
        ?string $description = null,
```

Also for the other parameters please